src/components/common/UserArticleCardComponent.js

```diff
@@ -1,14 +1,27 @@
 import React, { Fragment } from 'react';
 import { NavLink } from 'react-router-dom';
+import swal from 'sweetalert';
```

Comment:
I thought we were no longer using sweetalert?

Reply:
I am using it to display a warning modal before you delete an article